qemu/tests/device_option_check.py

```diff
@@ -109,6 +110,18 @@ def run(test, params, env):
                            " Guest output is '%s'" % (params_name,
                                                       output))
 
+        if sg_vpd_cmd:
+            error_context.context("Check serial number length with command %s"
+                                  % sg_vpd_cmd, logging.info)
+            sg_vpd_cmd = utils_misc.set_winutils_letter(session, sg_vpd_cmd)
+            output = session.cmd_output(sg_vpd_cmd)
+            act_len = output.split('bytes')[1].split()[-1]
+            act_len = int(act_len)
```

Comment:
```suggestion
            actual_len = int(output.split('bytes')[1].split()[-1])
```
Better to make variable `act_len` name meaningful, so suggest `actual_len`

Reply:
Updated, thanks

(1/1) Host_RHEL.m8.u2.product_av.qcow2.virtio_blk.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.device_option_check.blk_serial_check.q35: PASS (46.25 s)